api/src/test/java/io/druid/data/input/impl/JSONParseSpecTest.java

```diff
@@ -79,6 +79,31 @@ public void testParseRow()
     Assert.assertNull(parsedRow.get("path_omg2"));
   }
 
+  @Test
+  public void testParseRowWithConditional()
+  {
+    final JSONParseSpec parseSpec = new JSONParseSpec(
+        new TimestampSpec("timestamp", "iso", null),
+        new DimensionsSpec(DimensionsSpec.getDefaultSchemas(ImmutableList.of("foo")), null, null),
+        new JSONPathSpec(
+            true,
+            ImmutableList.of(
+                new JSONPathFieldSpec(JSONPathFieldType.PATH, "foo", "$.maybe_object.foo")
```

Comment:
> Add a unit test containing a JSONPath conditional

Did you mean to use a conditional in the path here? I don't think `$.maybe_object.foo` has one.